gensim/models/rpmodel.py

```diff
@@ -5,41 +5,54 @@
 # Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
 
 
-import logging
+"""
+Objects of this class allow building and maintaining a model for Random Projections
+(also known as Random Indexing). 
 
-import numpy as np
+For theoretical background on RP, see:   Kanerva et al.: "Random indexing of text samples for Latent Semantic Analysis."
 
-from gensim import interfaces, matutils, utils
+    The main methods are:
 
+    1. constructor, which creates the random projection matrix
+    2. the [] method, which transforms a simple count representation into the TfIdf
+       space.
 
-logger = logging.getLogger('gensim.models.rpmodel')
+Model persistency is achieved via its load/save methods.
 
 
-class RpModel(interfaces.TransformationABC):
-    """
-    Objects of this class allow building and maintaining a model for Random Projections
-    (also known as Random Indexing). For theoretical background on RP, see:
+Examples:
+---------
+>>> from gensim.models import rpmmodel
+>>> rp = RpModel(corpus)
```

Comment:
An example isn't executable (i.e. corpus isn't defined), "executable" means that I can copy-paste it to console and this runs successfully.